src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -1259,6 +1259,46 @@ class G1MergeHeapRootsTask : public WorkerTask {
     G1MergeCardSetStats stats() const { return _stats; }
   };
 
+  // Closure to clear the prev bitmap for any old region in the CSet. This is
```

Comment:
I would prefer if terms like `CSet` are not abbreviated in comments, i.e. s/CSet/collection set.

Reply:
I agree.